apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/FragmentsResponseMapperBuilder.kt

```diff
@@ -69,7 +69,7 @@ class FragmentsResponseMapperBuilder(
   private fun mapperFields(fragmentFields: List<ResponseFieldSpec>): List<FieldSpec> {
     return fragmentFields.map { field ->
       val fieldType = field.normalizedFieldSpec.type as ClassName
-      val mapperClassName = ClassName.get(context.ir.fragmentsPackageName, fieldType.simpleName(),
+      val mapperClassName = ClassName.get(fieldType.packageName(), fieldType.simpleName(),
```

Comment:
I think this should work but would love another pair of eyes.

Reply:
Looks good to me 👀